Misha/Files/zip_creator/zip_test.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import os
+from zip_main import check_set_of_paths, create_archive
+import zipfile
+
+
+class ZipTest(unittest.TestCase):
+
+    def test_check_set_of_paths_with_wrong_paths(self):
+        set_of_paths = ['path/to/file.1', 'path.txt', 'file.1']
+        with self.assertRaises(FileNotFoundError) as context:
+            self.assertRaises(FileNotFoundError, check_set_of_paths(set_of_paths))
```

Comment:
You have double assertion here. Please remove one

Reply:
i will correct

Reply:
corrected